spec/deps/git_test.go

```diff
@@ -42,26 +54,28 @@ func TestParseGit(t *testing.T) {
 			wantRemote: "https://github.com/ksonnet/ksonnet-lib",
 		},
 		{
-			name: "SSH",
-			uri:  "git+ssh://git@my.host:user/repo.git/foobar@v1",
-			want: &Dependency{
-				Version: "v1",
-				Source: Source{GitSource: &Git{
-					Scheme: GitSchemeSSH,
-					Host:   "my.host",
-					User:   "user",
-					Repo:   "repo",
-					Subdir: "/foobar",
-				}},
-			},
-			wantRemote: "ssh://git@my.host:user/repo.git",
```

Comment:
Should you keep a non github test ?

Reply:
Yes. For sure. I use it with a privately hosted gitea.io instance for example. GitLab might not even be as unpopular.